src/expander.rs

```diff
@@ -56,11 +56,13 @@ pub enum InvoiceVersioning {
 }
 
 impl InvoiceVersioning {
-    pub fn parse(text: &str) -> Self {
-        if text == "production" {
-            InvoiceVersioning::Production
+    pub fn parse(text: &str) -> anyhow::Result<Self> {
+        if text == "production" || text == "prod" {
+            Ok(InvoiceVersioning::Production)
+        } else if text == "development" || text == "dev" {
+            Ok(InvoiceVersioning::Dev)
         } else {
-            InvoiceVersioning::Dev
+            Err(anyhow::anyhow!("Invalid invoice versioning strategy: choose 'dev' or 'production'"))
```

Comment:
This is only a backstop in case Clap validation gets out of sync with what's here.  Longer term would be good to establish a single source of truth.